src/web/health_handler.go

```diff
@@ -0,0 +1,26 @@
+package web
+
+import (
+	"time"
+	"github.com/gin-gonic/gin"
+	"github.com/siesgstarena/epicentre/src/services/logger"
+)
+
+type health struct {
+    Name     		string `json:"name"`
```

Comment:
run `go fmt ./...` inside `src`

Reply:
This Indentation error of health also is not visible in my machine